Configuration/config.py

```
@@ -0,0 +1,16 @@
+# firebase admin 
+import firebase_admin
+# credentials,auth,firestore 
+from firebase_admin import credentials,auth,firestore 
```

Flake8 (7.2.0)

[error] 'firebase_admin.auth' imported but unused

(F401)